### PR TITLE
Feature/design

### PR DESCRIPTION
**Pull request by thisisui**

### CODE REVIEW

File: cogboard-webapp/src/components/EditWidget.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ EditWidget.propTypes = {
   goNewLine: bool.isRequired,
   id: string.isRequired,
   rows: number.isRequired,
-  title: string.isRequired,
+  title: string,
```
Comment (tMaxx) [inline cogboard-webapp/src/components/EditWidget.js]: Is this change intentional?

Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/EditWidget.js]: yes - Title should support to be empty
